### PR TITLE
Removed repos container

### PR DESCRIPTION
**Pull request by rohankeskarwednesday**

### CODE REVIEW

File: containers/Songs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      setSearchTerm(value);
+    }
+  }, 500);
+
+  return (
+    <Container
+      padding={20}
+      maxwidth={500}
+      style={{
+        height: "100vh",
```
Comment (christin-wednesday) [inline containers/Songs/index.tsx:43]: Please add a shimmer while the tracks are loading.
What happens if the API call fails ?

File: containers/Songs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        alignSelf: "center",
+      }}
+    >
+      <Search placeholder="Search for songs" onChange={handleOnChange} />
```
Comment (christin-wednesday) [inline containers/Songs/index.tsx]: Please add internationlization support. Later we can add support for Hindi.

File: features/songs/api/getItunesSongs.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+type Params = {
+  searchTerm: string;
+};
+
+export const songsApi = itunesApiService.injectEndpoints({
+  endpoints: builder => ({
+    fetchSongs: builder.query<RepoInfoResponse, object>({
+      query: (params: Params) => {
+        if (!params.searchTerm) {
+          throw new Error("Invalid params");
```
Comment (christin-wednesday) [inline features/songs/api/getItunesSongs.ts:14]: 🚀

File: store/slices/songs.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  reducers: {
+    successGetRepos: (state: SongState, action: PayloadAction<any[]>) => {
+      state.songs = action.payload;
+      state.songsCount = 0;
+    },
+  },
+});
+
+export const { successGetRepos } = songsSlice.actions;
+
```
Comment (christin-wednesday) [inline store/slices/songs.ts:28]: Please write tests for this reducer.

**Review by christin-wednesday — CHANGES_REQUESTED** [review]
Please deploy this on vercel / or any hosting platform that you prefer

File: containers/Songs/index.tsx (hunk 1 above)
Comment (rohankeskarwednesday) [reply containers/Songs/index.tsx:39]: Haven't added condition for api fail, Will add a error message

Comment (rohankeskarwednesday) [reply containers/Songs/index.tsx:43]: > Please add a shimmer while the tracks are loading. What happens if the API call fails ?

Haven't added condition for api fail, Will add a error message

**Pull request merged**